python/pyglome/glome.py

```diff
@@ -127,9 +126,10 @@ def __init__(self,
         else:
             my_public_key = my_private_key.public_key()
 
-        if not Glome.MIN_TAG_LEN < min_peer_tag_len <= Glome.MAX_TAG_LEN:
-            raise ValueError('min_peer_tag_len must be in range {}-{}'.format(
-                Glome.MIN_TAG_LEN, Glome.MAX_TAG_LEN))
+        if not Glome.MIN_TAG_LEN <= min_peer_tag_len <= Glome.MAX_TAG_LEN:
+            raise ValueError(
+                f'min_peer_tag_len (={min_peer_tag_len}) is not within '
+                f'({Glome.MIN_TAG_LEN}; {Glome.MAX_TAG_LEN}]')
```

Comment:
Should the left brace be a bracket?

Reply:
Yes, good catch. I think I've updated the message first, and only then spotted the comparison bug. And obviously forgot to update the message again.